### PR TITLE
Add more detailed instructions on how to build Slycot

### PR DESCRIPTION
**Pull request by sdrave**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +95,33 @@ because they need additional setup on your system:
 
       pip install slycot
 
+  Note that building Slycot might fail for the following reasons:
+  * The Slycot package contains a cmake check which fails when it
```
Comment (pmli) [inline README.md:99]: Perhaps there should be an empty line above the start of a list.

File: README.md [hunk 2]
```diff
@@ -95,6 +95,33 @@ because they need additional setup on your system:
...
+    detects multiply NumPy include directories. This will cause the
+    build to fail in venvs with any Python interpreter that has NumPy
+    globally installed.
+    To circumvent this problem, use another Python interpreter. If
+    you do not want to build CPython yourself, you can use
+    [pyenv](https://github.com/pyenv/pyenv) or
+    [uv](https://github.com/astral-sh/uv) to easily install another
+    interpreter.
```
Comment (pmli) [inline README.md:107]: [mise](https://mise.jdx.dev/) could also be suggested.

File: README.md [hunk 3]
```diff
@@ -95,6 +95,33 @@ because they need additional setup on your system:
...
+    On most systems, an older NumPy version is installed, whose
+    f2py will generate incorrect wrapper code for `numpy>=2`.
+    To mitigate this issue, install `numpy>=2` into your venv
+    and link `f2py3` to `f2py` its `/bin` directory.
+  * Building Slycot on Windows is challenging. We recommend using
+    conda-forge packages instead. If you do not want to install
+    the pyMOR conda-forge package, you can also `pip` install pyMOR
+    into an existing conda environment.
+
+  If you are on linux and don't want to build Slycot yourself, you
```
Comment (pmli) [inline README.md:121]:
```suggestion
  If you are on Linux and don't want to build Slycot yourself, you
```

**Review by pmli — APPROVED** [review]
Only minor comments, can be ignored.

File: README.md (hunk 1 above)
Comment (sdrave) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (sdrave) [reply README.md]: Done.

**Pull request merged**